shared/actions/platform-specific/push.native.js

```diff
@@ -127,8 +127,9 @@ function* handleLoudMessage(notification) {
 
   const {conversationIDKey, unboxPayload, membersType} = notification
 
+  yield Saga.put(RouteTreeGen.createSwitchTab({tab: 'tabs.chatTab'}))
+  yield Saga.put(RouteTreeGen.createNavUpToScreen({routeName: 'chatRoot'}))
```

Comment:
There's a delay on the previously selected convo before it switches over to the new one (too long in dev, on a profile build it's not too bad). I think fine for now, but maybe switching this to do a `reset` later instead would help